cpp/test/test.cpp

```diff
@@ -42,6 +48,30 @@ std::string vec_to_string(RustVeci8 a)
     return std::string(reinterpret_cast<char const *>(&a[0]), a.size());
 }
 
+std::string random_string(std::size_t length)
+{
+    const std::string CHARACTERS = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz";
+
+    std::random_device random_device;
+    std::mt19937 generator(random_device());
+    std::uniform_int_distribution<> distribution(0, CHARACTERS.size() - 1);
+
+    std::string random_string;
+
+    for (std::size_t i = 0; i < length; ++i)
+    {
+        random_string += CHARACTERS[distribution(generator)];
+    }
+
+    return random_string;
+}
+std::string random_id()
+{
+    return random_string(94);
+}
+
+auto deviceContextString = "{\"accountId\":\"abcABC012_.$#|@/:;=+'-91e078f0-a60c-4251-8652-dd498c07a8f4\",\"segmentId\":1825,\"signingPrivateKey\":\"uKHa70uwLVG3IU7XodT2kla/PuC/En8PkRCjMMc9ZE7HFrOV+g0vOwATp/CiXp65mVas0K6TSl/RaxDGlcmsnA==\",\"devicePrivateKey\":\"YZRlDSkM+JxxSXCtWCVK693qfhNqcbhaPrtHs92uD4w=\"}";
```

Comment:
Where is this device context from? We've recently been trying to be conscious about what devices we make public, so just checking

Reply:
I stole it from an ironoxide unit test. If you have the ability to generate one for the same segment we use for ironoxide-android, we should just use one of those instead.